alinka/widget/containers/main_body/footer/settings/support_center_tab.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, parent: QWidget, visible: bool = False):
 
     @property
     def support_center_data(self) -> SupportCenterData:
-        return self.parent.parent.parent.content_container.settings_container.support_center_data
+        return self.parent().parent().parent().content_container.settings_container.support_center_data
```

Comment:
That's kind of surprise. With update to `PySide6` also api has changed for widgets. I have propositions:
* either use method from [docs](https://doc.qt.io/qtforpython-6/PySide6/QtWidgets/QWidget.html#PySide6.QtWidgets.QWidget.parentWidget) (`parentWidget`)
* or better create property for given widget which refers parent, but has more meaningful name:
```
@property
def main_container(self):
    return self.parentWidget()
```
This can be covered by another ticket.